src/users/users.controller.ts

```diff
@@ -104,7 +107,7 @@ export class UsersController {
         description: "This is currently only for development/admin",
     })
     @ApiResponse({
-        status: HttpStatus.OK,
+        status: HttpStatus.CREATED,
```

Comment:
this one should be OK, created is usually used when a resource is created and will return 201

Reply:
oh. I know why. By default all POSTs are response 201 in nest.js, when you changed from GET to POST it gets updated automatically

https://docs.nestjs.com/controllers#resources
![image](https://github.com/chingu-x/chingu-dashboard-be/assets/6191116/9ac62772-ba30-4147-ba09-8de0512490e4)